### PR TITLE
Remove config groups through liblepton configuration API

### PR DESCRIPTION
**Pull request by graahnul-grom**
Extend `liblepton` configuration API by adding `eda_config_remove_group()`
function (`config-remove-group!` in Scheme).
The `group` parameter of the `config-changed` signal is set to the group name
deleted and the `key` parameter is set to an empty string.

Related to PR #294 where `eda_config_remove_key()` and `config-remove-key!`
functions were added.

### CODE REVIEW

File: docs/scheme-api/.gitignore [hunk 1]
```diff
@@ -2,3 +2,4 @@
 lepton-scheme.info
 stamp-vti
 version.texi
+lepton-scheme.html/*
```
Comment (vzh) [inline docs/scheme-api/.gitignore]: This is superfluous here. (BTW, what command should be issued to make the directory, `make html`?). Please put it into a separate commit with appropriate description.
Anything other suits me well, thank you!

Comment (vzh) [reply docs/scheme-api/.gitignore]: Ah, OK, just checked it.

Comment (graahnul-grom) [reply docs/scheme-api/.gitignore]: I use these commands in my install script:

`cd docs/scheme-api/`
`make html`
`make install-html`
